components/mpas-ocean/src/Registry.xml

```diff
@@ -2867,6 +3067,27 @@
 		<var name="indMLD" type="integer" dimensions="nCells Time" units="unitless"
 			description="index of model where mixed layer depth occurs (always one past)"
 		/>
+                <var name="ugrad" type="real" dimensions="R3 nVertLevels nCells Time" units="s^{-1}"
+                        description="three-dimensional gradient of zonal velocity"
+                />
+                <var name="vgrad" type="real" dimensions="R3 nVertLevels nCells Time" units="s^{-1}"
+                        description="three-dimensional gradient of meridional velocity"
+                />
+                <var name="wgrad" type="real" dimensions="R3 nVertLevels nCells Time" units="s^{-1}"
+                        description="three-dimensional gradient of vertical velocity"
+                />
+                <var name="tgrad" type="real" dimensions="R3 nVertLevels nCells Time" units="s^{-1}"
+                        description="three-dimensional gradient of temperature"
+                />
+                <var name="sgrad" type="real" dimensions="R3 nVertLevels nCells Time" units="s^{-1}"
+                        description="three-dimensional gradient of salinity"
+                />
+                <var name="Sh" type="real" dimensions="nVertLevels nCells Time" units="unitless"
+                        description="Structure function for heat and salt -- may want to consider separating at some point"
+                />
+                <var name="Sm" type="real" dimensions="nVertLevels nCells Time" units="unitless"
+                        description="Structure function for momentum"
+                />
```

Comment:
I'm worried that the names `Sh` and `Sm` are not sufficiently descriptive or unique.